app/app.py

```diff
@@ -134,6 +136,15 @@ def _handle_premis_event(event: PremisEvent):
             fragment_id=event.fragment_id,
             pid=event.external_id
         )
+        # Send a message to an "error" exchange for reporting purposes
+        routing_key = config.config["environment"]["rabbit"]["routing_key_nok"]
+        exchange = config.config["environment"]["rabbit"]["exchange_nok"]
+        RabbitService(config=config.config).publish_message(event.to_string(), exchange, routing_key)
```

Comment:
`RabbitService` is instantiated twice in this function: might be better to move this into the "global scope" in this file (of course, only once on every function call, but still). Especially, because it opens and closes a new connection every time (see: https://www.cloudamqp.com/blog/2018-01-19-part4-rabbitmq-13-common-errors.html)
More broadly, we need to think about the connections an application uses during it's lifetime and how to re-use them.
(will be new issue)